styles.css

```diff
@@ -144,17 +149,32 @@ img {
   }
 
   .header-text {
-    font-size: 18px;
-    padding: 10px;
+    font-size: 16px;
+    font-weight: 900;
+    /* padding: 10px; */
+    display: inline;
+    margin-right: 8.2em;
+    }
+
+  .hamburger-button {
+    background-color: transparent;
+    border: none;
+    margin-right: 5.5em;
+    width: 3em;
+    height: 2.5em;
```

Comment:
When we go to refactor this probably try and stray away from hard values for margins and padding, though it is the header so not the biggest deal.